### PR TITLE
Placeholder content for the TSP/Public API

### PR DESCRIPTION
**Pull request by ntwyman**
## Description

Beginning to populate out the api.yaml with a couple of example TSP related endpoints. 

## Reviewer Notes

The definition of these endpoints is incomplete/not final & they are NotImplemented yet internally. That said the code is all hooked up and you can go to the docs /api/v1/docs and see the definitions. You can also Execute requests from the docs but you will get a 501 error back

## Verification Steps

* [x] All tests pass.
* [n/a] The requirements listed in [Querying the Database Safely](https://github.com/transcom/mymove/blob/master/docs/backend.md#querying-the-database-safely) have been satisfied.
* [n/a] There are no [aXe](https://www.deque.com/products/aXe/) warnings for UI.
* [n/a] This works in IE.

## References

* [Produce straw man TSP API documentation/workflow based docs sent to TSPs](https://www.pivotaltracker.com/story/show/155009577) for this change

### CODE REVIEW

**Comment by ntwyman** [comment]
Sadly - have not yet succeeded in shaving the split YAML file yak:
 * theres a bug in the go-swagger yaml I hit
* I ran into issues getting relative yaml references to work consistently at build time and runtime which I am sure are manageable but I don't want to hold up this code.

File: swagger/api.yaml [hunk 1]
```diff
@@ -19,10 +19,377 @@ paths:
...
+        200:
+          description: list of associated shipments
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/Shipment'
+        400:
+          description: invalid request
+        403:
+          description: not authorized to get list of TSPs
```
Comment (jim) [inline swagger/api.yaml]: This should be `not authorized to get list of shipments`.

File: swagger/api.yaml [hunk 2]
```diff
@@ -19,10 +19,377 @@ paths:
...
+            type: array
+            items:
+              $ref: '#/definitions/TSP'
         400:
           description: invalid request
+        403:
+          description: not authorized to get list of TSPs
+  /tsp/{tspUUID}/shipments:
+    get:
+      summary: Retreive shipments associated with a TSP
```
Comment (jim) [inline swagger/api.yaml]: s/Retreive/Retrieve

File: swagger/api.yaml [hunk 3]
```diff
@@ -19,10 +19,377 @@ paths:
...
+    type: object
+    description: The primary definition of a Transport Service Provider
+    properties:
+      id:
+        type: string
+        format: uuid
+        example: c56a4180-65aa-42ec-a945-5fd21dec0538
+      name:
+        type: string
+        example: Name of the TSP
```
Comment (jim) [inline swagger/api.yaml]: Might be better to have something realistic here, e.g. `Moving Company, Inc.`.

File: swagger/api.yaml [hunk 4]
```diff
@@ -19,10 +19,377 @@ paths:
...
+          - US49
+        example: US49
+      destination:
+        type: string
+        enum: # Add other values
+          - REGION_6
+        example: REGION_6
+  PreMoveSurvey:
+    type: object
+    description: A shipment which may be awarded to a TSP
```
Comment (jim) [inline swagger/api.yaml]: This might be ok, but this `description` doesn't make it obvious to me what a `PreMoveSurvey` is.

**Pull request merged**